export const a = () => { throw "kaboom"; };
                         ~~~~~~~~~~~~~~~                [throw-error]
export const b = () => { throw new Error("kaboom"); };
export const c = () => { throw new DOMException("kaboom"); };

try {
    throw new Error("kaboom");
} catch (error: any) {
    throw error;
}

try {
    throw new DOMException("kaboom");
} catch (error: any) {
    throw error;
}

function throwError(error: any): never {
    throw error;
}

[throw-error]: Throwing non-Error values is forbidden
